UM/Settings/ContainerRegistry.py

```diff
@@ -166,36 +167,58 @@ def addContainerType(cls, container):
     #   If this function is called again, it will clear the old data and reload.
```

Comment:
You might want to expand the documentation of this function a bit. I think the last sentence is not entirely correct any more. It will reload, but it will reload from the cache anyway...